dojo/templates/dojo/view_finding.html

```diff
@@ -473,48 +473,64 @@ <h4>Similar Findings<span class="pull-right"><a data-toggle="collapse" href="#si
                 <div id="similar_findings" class="panel-body collapse in">
                     <table class="table table-striped table-hover centered">
                         <tr>
-                            <th>Finding</th>
+                            <th>Title</th>
                             <th>Status</th>
-                            <th>Tags</th>
+                            <th>Date</th>
                             <th>Test</th>
                             <th>Engagement</th>
-                            <th>Version</th>
+                            {% if finding.cwe > 0 %}
+                            <th>CWE</th>
+                            {% endif %}
                             <th>CVE</th>
                             <th>File</th>
                             <th>Deduplicate?</th>
                         </tr>
                         {% for similar in findings %}
                             <tr>
+                                <td>
+                                    <a title="{{ similar.title }}" href="{% url 'view_finding' similar.id %}">{{ similar.title|truncatechars_html:80 }}</a>
+                                    {% if similar.tags %}
+                                    <small>
+                                        {% for tag in similar.tags %}
+                                            <a title="Search {{ tag }}" class="tag-label tag-color"
+                                               href="{% url 'simple_search' %}?query={{ tag }}">{{ tag }}</a>
+                                        {% endfor %}
+                                    </small>
+                                    {% endif %}
+                                </td>
+                                <td>{{ similar.status }}{% if similar.duplicate_finding_id %}, <a href="{% url 'view_finding' similar.duplicate_finding_id %}">Original</a> {% endif %}</td>
                                 <td>
                                     <a target="#" data-toggle="tooltip" data-placement="bottom" title="{{ similar.title }}" href="{% url 'view_finding' similar.id %}">{{ similar.date }}</a>
```

Comment:
Can probably be removed, since we have a whole column to bring about that data now?

Reply:
Not sure I understand, what part do you think can be removed?

Reply:
The whole hover part. What is displayed is essentially what's now in the new column `Title`

Reply:
ah, ok nice catch. And quite special that all lines are made green by github, except the one you are referring to haha

Reply:
should be fixed in next commit